examples/js/effects/VREffect.js

```diff
@@ -141,17 +134,35 @@ THREE.VREffect = function ( renderer, onError ) {
 				eyeWidth = eyeParamsL.renderWidth;
 				eyeHeight = eyeParamsL.renderHeight;
 
+				if ( vrDisplay.getLayers ) {
+
+					var layers = vrDisplay.getLayers();
+					if (layers.length) {
+
+						leftBounds = layers[0].leftBounds || [ 0.0, 0.0, 0.5, 1.0 ];
```

Comment:
Thanks, @toji. I don't see any reason to change this. The spec is very clear about what the default values are, so if they're not specified you can still know what they are.

Though I do feel the layer object should be a copy of the original so nothing gets screwed up if someone changes the values inside.